wgpu-core/src/instance.rs

```diff
@@ -493,6 +494,9 @@ impl<G: GlobalIdentityHandlerFactory> Global<G> {
                 hal::adapter::DeviceType::VirtualGpu => {
                     virt = virt.or(Some(i));
                 }
+                hal::adapter::DeviceType::Cpu => {
+                    cpu = cpu.or(Some(i));
+                }
                 _ => {
```

Comment:
let's list this variant explicitly as `hal::adapter::DeviceType::Other`

Reply:
done